sktime/transformations/multiplexer.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Implements forecaster for selecting among different model classes."""
```

Comment:
transformer

Reply:
also not "model classes" but "estimators", no?

Reply:
Yes, good spots here, and elsewhere!